scripts/test-build-install-zip-file.sh

```diff
@@ -1,17 +1,22 @@
 #!/bin/bash
 
-set -euxo pipefail
+set -euo pipefail
```

Comment:
Removed `set -x` from `scripts/test-build*.sh` shell scripts.
It looks to me that it is not needed at least in these targets.

I'm not totally sure if this flag should be kept in the other scripts (e.g. `scripts/test-check-packages.sh`).

WDYT ? Should we remove `set -x` in all these scripts ?